openshift_scalability/config/golang/nodeVertical-labeled-nodes.yaml

```diff
@@ -0,0 +1,21 @@
+provider: local
+ClusterLoader:
+  cleanup: true
+  projects:
+    - num: 1
+      basename: clusterproject
+      tuning: default
+      ifexists: delete
+      pods:
+        - num: 1000
```

Comment:
Do we want to keep default number of pods at 500 aka 250 pods per node ?

Reply:
We pick 4 app nodes, so assuming 250 pods/node, we need to create 1000 pods in total.